c_src/trans_nif.c

```diff
@@ -116,8 +115,18 @@ confidence(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
   if (! (target = (decimal*) enif_make_new_binary(env, count * sizeof(decimal), &r)))
     return enif_make_badarg(env); // TODO return propper error
   for (int i = 0; i < count; i++) {
-    target[i] = temp;
-    target[i].coefficient = vs[i].confidence;
+    if (vs[i].confidence != 0)
+      target[i] = (decimal){
+        .coefficient = vs[i].confidence,
+        .exponent = -3,
+        .confidence = CERTAIN
+      };
+    else
+      target[i] = (decimal){
+        .coefficient = 0,
+        .exponent = 0,
```

Comment:
is .coefficent = 0 , .exponen = -3 going to cause a problem or is there another reason to expand this in two defintions?